include/inviwo/core/datastructures/histogram.h

```diff
@@ -34,10 +34,15 @@
 
 #include <iterator>
 #include <vector>
+#include <bitset>
 
 namespace inviwo {
 enum class HistogramMode { Off, All, P99, P95, P90, Log };
 
+using HistogramSelection = std::bitset<32>;
```

Comment:
32? Does it mean that 32 channels are supported?

Reply:
Yes it can handle turning on and off 32 different histograms at the same time. The histograms can represent different channels but the are not limited to that, might also be a set of different volumes etc. wanted to avoid needing to use a full vector<bool> at this point...